.github/workflows/integration.yml

```diff
@@ -238,7 +238,7 @@ jobs:
 
     - name: Uninstall Orbit
       run: |
-        ./orbit/tools/cleanup/cleanup_macos.sh
+        sudo ./orbit/tools/cleanup/cleanup_macos.sh
```

Comment:
noted this in slack, over-communicating here that there's an alternative fix https://github.com/fleetdm/fleet/pull/20226/files

I wouldn't change it so you don't lose all the reviews